### PR TITLE
[NTOS:OB] ObQueryDeviceMapInformation(): Fix annotations

### PR DESCRIPTION
**Pull request by SergeGautherie**
Be consistent and correct...

### CODE REVIEW

File: ntoskrnl/ob/devicemap.c [hunk 1]
```diff
@@ -536,9 +536,10 @@ ObInheritDeviceMap(IN PEPROCESS Parent,
 
 NTSTATUS
 NTAPI
-ObQueryDeviceMapInformation(IN PEPROCESS Process,
-                            IN PPROCESS_DEVICEMAP_INFORMATION DeviceMapInfo,
-                            IN ULONG Flags)
+ObQueryDeviceMapInformation(
+    _In_opt_ PEPROCESS Process,
+    _Out_ PPROCESS_DEVICEMAP_INFORMATION DeviceMapInfo,
+    _In_ ULONG Flags)
```
Comment (GeoB99) [inline ntoskrnl/ob/devicemap.c:542]: Are you intending to introduce SAL annotations for the whole file or just that function?

Comment (SergeGautherie) [reply ntoskrnl/ob/devicemap.c:542]: This function only, for the time being. (Noticed while checking #3634.)

Comment (Extravert-ir) [reply ntoskrnl/ob/devicemap.c:542]: What's the point in annotating only one function in a file?

Comment (SergeGautherie) [reply ntoskrnl/ob/devicemap.c:542]: It's a start... Does that cause any harm?

**Comment by GeoB99** [comment]
> It's a start... Does that cause any harm?

This time I'll give it a pass but in the future you can make a PR that annotates the functions with SAL of the whole file instead of making million of PRs (and by the way, you have like 44 open PRs left that you must address yourself, don't ya think?)

**Pull request merged**